doc/dnf5.conf.5.rst

```diff
@@ -92,7 +92,10 @@ repository configuration file should aside from repo ID consists of baseurl, met
 ``cachedir``
     :ref:`string <string-label>`
 
-    Path to a directory used by various DNF5 subsystems for storing cache data.
+    Path to a directory used by various DNF5 subsystems for storing cache data
+    including repositories metadata and downloaded packages. Repositories cache
+    path is using unique hash computed from substituted URLs (metalink, mirrorlist
```

Comment:
I find this addendum confusing. You want to describe what happens under `cachedir` path. But a reader might confuse the `cachedir` path with your "Repositories cache path" and get an impression that he is expected to write some hashes into the configuration value.

I think documenting internals how the cache directory content is organized, especially if the internals are not a public interface and can change anytime, here in dnf5.conf(5) manual wrong. If you want to document the unstable, nonpublic internals, do it in in a separate developer-oriented manual. Not in a manual of public and stable interface targeting end users.

Reply:
Good point, I am rephrasing the description